ecs/cloudwatch.conf

```diff
@@ -0,0 +1,11 @@
+@INCLUDE /ecs/service.conf
+@INCLUDE /ecs/input-forward.conf
+@INCLUDE /ecs/ecs-metadata.conf
+
+[OUTPUT]
+    Name   cloudwatch_logs
+    Match  *
+    region us-east-1
```

Comment:
I just realized region needs to be parametrized too...

Let's say that the init process also adds an env var `AWS_REGION`. Which you can use here. 

For CW, we assume folks want to send logs to the same region that they are running tasks in. For other outputs like firehose, while they should use the same region, they don;t have to, hence we need to make the region come from ther user.